doc/getting_started/installation_win.rst

```diff
@@ -71,117 +83,11 @@ packages from their respective websites.
-      of the ``--user`` flag, this makes it harder for the Command Prompt
-      to find the executables in Python modules installed by ``pip3``.
+      choco install git python ninja dtc-msys2 gperf
 
-#. The build system should now be ready to work with any toolchain installed in
-   your system. In the next step you'll find instructions for installing
-   toolchains for building both x86 and ARM applications.
-
-#. Install cross compiler toolchain:
-
-   * For x86, install the 2017 Windows host ISSM toolchain from the Intel
-     Developer Zone: `ISSM Toolchain`_. Use your web browser to
-     download the toolchain's ``tar.gz`` file. You can then use 7-Zip or a
-     similar tool to extract it into a destination folder.
-
-     .. note::
-
-        The ISSM toolset only supports development for Intel |reg| Quark |trade|
-        Microcontrollers, for example, the Arduino 101 board.  (Check out the
-        "Zephyr Development Environment
-        Setup" in this `Getting Started on Arduino 101 with ISSM`_ document.)
-        Additional setup is required to use the ISSM GUI for development.
-
-
-   * For ARM, install GNU ARM Embedded from the ARM developer website:
-     `GNU ARM Embedded`_ (install to :file:`c:\\gnuarmemb`).
-
-#. Within the Command Prompt, set up environment variables for the installed
-   tools and for the Zephyr environment:
-
-   For x86:
-
-   .. code-block:: console
-
-      set ZEPHYR_TOOLCHAIN_VARIANT=issm
-      set ISSM_INSTALLATION_PATH=c:\issm0-toolchain-windows-2017-01-25
-
-   Use the path where you extracted the ISSM toolchain.
-
-   For ARM:
-
-   .. code-block:: console
-
-      set ZEPHYR_TOOLCHAIN_VARIANT=gnuarmemb
-      set GNUARMEMB_TOOLCHAIN_PATH=c:\gnuarmemb
-
-   To use the same toolchain in new sessions in the future you can set the
-   variables in the file :file:`%userprofile%\\zephyrrc.cmd`.
-
-   And for either, run the :file:`zephyr-env.cmd` file in order to set the
-   :makevar:`ZEPHYR_BASE` environment variable:
-
-   .. code-block:: console
+#. Close the Administrator command prompt window.
 
-      zephyr-env.cmd
-
-   .. note:: In previous releases of Zephyr, the ``ZEPHYR_TOOLCHAIN_VARIANT``
-             variable was called ``ZEPHYR_GCC_VARIANT``.
-
-#. Finally, you can try building the :ref:`hello_world` sample to check things
-   out.
-
-   To build for the Intel |reg| Quark |trade| (x86-based) Arduino 101:
-
-   .. zephyr-app-commands::
-     :zephyr-app: samples/hello_world
-     :host-os: win
-     :generator: ninja
-     :board: arduino_101
-     :goals: build
-
-   To build for the ARM-based Nordic nRF52 Development Kit:
-
-   .. zephyr-app-commands::
-     :zephyr-app: samples/hello_world
-     :host-os: win
-     :generator: ninja
-     :board: nrf52_pca10040
-     :goals: build
-
-This should check that all the tools and toolchain are set up correctly for
-your own Zephyr development.
+.. _windows_install_msys2:
 
 Option 2: MSYS2
```

Comment:
Let's use this opportunity to reorder MSYS2 to the bottom, after WSL. We really do not want to have to support this in a platform we already have native support for, and bash-type access is provided by WSL.

Reply:
Sounds very reasonable especially since WSL is supported by the OS vendor.